### PR TITLE
PLAT-24238: Add permission to block new direct login users

### PR DESCRIPTION
**Pull request by drorsou**

### CODE REVIEW

File: admin_console/forms/PartnerConfiguration.php [hunk 1]
```diff
@@ -591,6 +592,11 @@ public function init()
 			'decorators' => array('ViewHelper', array('Label', array('placement' => 'append')), array('HtmlTag',  array('tag' => 'dt', 'class' => 'partner_configuration_checkbox_field_only')))
 		));
 		
+		$this->addElement('checkbox', 'ALLOW_SSO_PER_USER', array(
```
Comment (inbal-ben-david) [inline admin_console/forms/PartnerConfiguration.php:595]: will prefer to have this in lowercase like the rest of the element names

File: admin_console/forms/PartnerConfiguration.php [hunk 2]
```diff
@@ -625,6 +631,10 @@ public function init()
 	
 			foreach($moduls as $name => $modul)
 			{
+				if ($modul->skip)
```
Comment (inbal-ben-david) [inline admin_console/forms/PartnerConfiguration.php:634]: if you skip you ignore the default decorator?

File: release-notes.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
+# Scorpius-19.10.0
+## Add permission to block new direct login users ##
```
Comment (inbal-ben-david) [inline release-notes.md:2]: i would rephrase it to Add permission to allow setting SSO per user
like the permission name you gave

File: admin_console/forms/PartnerConfiguration.php (hunk 1 above)
Comment (drorsou) [reply admin_console/forms/PartnerConfiguration.php:595]: Because the permission name on admin.ini is all caps, then here it is all caps.
So it will be an exception either be here as caps, or on admin.ini as lowercase.

**Review by inbal-ben-david — APPROVED** [review]
